recipes/molmod/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "molmod" %}
+{% set version = "1.4.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ca0e461fe42fc8b7e0b481cda33aac247d86b5fb817b5076e20abc90dd577797
+
+build:
+  number: 0
+  script: "{{ PYTHON }} setup.py install --single-version-externally-managed --record record.txt"
+  skip: True  # [not linux or py<37]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cython >=0.24.1
+  host:
+    - python
+    - pip
+    - cython >=0.24.1
+    - numpy >=1.0
+    - future
+    - setuptools
+    - gpaw
+  run:
+    - numpy >=1.0
+    - future
+    - gpaw
```

Comment:
`gpaw` is not a dependency, https://github.com/molmod/molmod/blob/master/setup.py#L96-L97

Reply:
It is not listed but it does not work without it - you can try - I basically got the information from private communication with the developers.